relengapi/blueprints/slaveloan/slave_mappings.py

```diff
@@ -108,3 +108,16 @@ def _inner_slave_filter(item):
                 return True
         return False  # If we got here, no match
     return _inner_slave_filter
+
+
+def slavetype_to_awsprefix(slave_class):
+    if not is_aws_serviceable(slave_class):
+        raise ValueError("Unsupported Slave")
+    basic_slave_prefix = slave_to_slavetype(slave_class)
+    if basic_slave_prefix.startswith("bld"):
+        loan_prefix = basic_slave_prefix.replace("bld-", "dev-")
+    elif basic_slave_prefix.startswith("tst"):
+        loan_prefix = basic_slave_prefix
+    else:
+        raise ValueError("Unsure how to name this aws slave")
+    return loan_prefix
```

Comment:
This is starting to look like so much other Releng code, all of which is a little bit different and has to be changed whenever a naming convention change occurs.  I'd really like to see this in configuration, rather than in code.